BenchmarkTests/Runner/Scenarios/Trace/UI/TraceContentView.swift

```diff
@@ -0,0 +1,173 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import DatadogTrace
+import SwiftUI
+
+struct TraceContentView: View {
+    @State private var operationName: String
+    @State private var resourceName: String
+    @State private var isError: Bool
+    @State private var depth: Int
+    @State private var childrenCount: Int
+    @State private var childDelay: TimeInterval
+    @State private var traceCount: Int
+    @State private var isSending: Bool
+
+    var tracer: OTTracer { Tracer.shared() }
+
+    private let queue1 = DispatchQueue(label: "com.datadoghq.benchmark-tracing1")
+    private let queue2 = DispatchQueue(label: "com.datadoghq.benchmark-tracing2")
+
+    init() {
+        operationName = "iOS Benchmark span operation"
+        resourceName = "iOS Benchmark span resource"
+        isError = false
+        depth = 1
+        childrenCount = 0
+        childDelay = 1
```

Comment:
/suggestion We should use milliseconds from more realistic traces, we could use 100ms by default with 50ms steps.